src/variable.jl

```diff
@@ -401,7 +401,7 @@ end
 getchunksize(v::Variable) = getchunksize(haschunks(v),v)
 getchunksize(::DiskArrays.Chunked, v::Variable) = chunking(v)[2]
 # getchunksize(::DiskArrays.Unchunked, v::Variable) = DiskArrays.estimate_chunksize(v)
-getchunksize(::DiskArrays.Unchunked, v::Variable) = size(v)
+getchunksize(::DiskArrays.Unchunked, v::Variable) = max.(1,size(v))
 eachchunk(v::CFVariable) = eachchunk(v.var)
 haschunks(v::CFVariable) = haschunks(v.var)
```

Comment:
Just to clarify this declaration on `CFVariable` doesn't really do much, as it doesn't define the necessary broadcast or iteration methods to actually call this in real use.